hc/api/tests/test_protocols.py

```diff
@@ -0,0 +1,67 @@
+from datetime import timedelta
+
+from django.utils import timezone
+from hc.api.models import Check, Channel
+from hc.test import BaseTestCase
+from mock import patch
+from django.contrib.auth.hashers import make_password
+
+
+def fake_twilio_notify():
```

Comment:
Any reason this is still here?

Reply:
I use it to mock the notify method in TwilioSms class. see line[21] below, since I am only testing my logic in check.send_alert().

Reply:
Ohh I see